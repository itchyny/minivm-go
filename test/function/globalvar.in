a = 10
b = 20
func foo(x)
  b = 30
  return a * b + x + c
end

c = 50
print foo(50)
print a
print b
