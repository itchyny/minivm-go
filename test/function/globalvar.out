400
10
20
